apps/meteor/app/lib/server/methods/getChannelHistory.ts

```diff
@@ -67,7 +68,9 @@ Meteor.methods<ServerMethods>({
 			throw new Meteor.Error('error-invalid-date', 'Invalid date', { method: 'getChannelHistory' });
 		}
 
-		const hiddenMessageTypes = getHiddenSystemMessages(room);
+		const hiddenSystemMessages = settings.get('Hide_System_Messages') as MessageTypesValues[];
```

Comment:
Curious: can you pass the type on the type parameter of settings.get? Otherwise just ignore this comment